modules/web/js/ballerina/diagram2/views/default/components/nodes/top-level-nodes.jsx

```diff
@@ -109,6 +126,19 @@ class PackageDeclarationNode extends React.Component {
             this.onPackageInputBlur(event);
         }
     }
+
+    /**
+     * Get the full package name
+     */
+    getPackageName(model) {
```

Comment:
Return value is missing from the documentation.

Reply:
Fixed with commit 03ae4b2